template/workflows/helm/.github/workflows/azure-kubernetes-service-helm.yml

```diff
@@ -21,13 +21,14 @@
 #    - AZURE_CONTAINER_REGISTRY (name of your container registry / ACR)
 #    - CONTAINER_NAME (name of the container image you would like to push up to your ACR)
 #    - CLUSTER_RESOURCE_GROUP (where your cluster is deployed)
-#    - CLUSTER_NAME (name of your AKS cluster)
```

Comment:
For backward compatibility, we should still use CLUSTER_NAME instead of changing to RESOURCE_NAME.  In the ARM API, we'll still be using clustername as well iirc.

Reply:
changed